### DISCUSSION TITLE
Ability to add views to default "auto generated" Overview dashboard

### DISCUSSION THREAD

**Discussion by charettepa** [home-assistant/frontend#11305] [Other feature requests]
## The request

Ability to add views to default (auto) dashboard
you would be able to add your own view(s) while maintaining the default "auto generated" Overview dashboard

## The alternatives

Ability to add the default dashboard as a view in a custom dashboard
This would allow to create a custom dashboard and make it default
but still have access to the default "auto generated" Overview dashboard which can be useful at times
     because, unfortunately, as soon as you make a custom dashboard default, the (Overview) disappears

I managed to add one myself by creating a view and changing the path to ../lovelace/default_view
     but this is risky as if you accidentally move that view to be first, you start a loop and only ever get the Overview

ability to add it as a view when creating the dashboard could place it in a fixed place not allowing to move it avoiding my issue

here is a screenshot of what I have now, something similar that cant accidentally move would be nice

![image](https://user-images.githubusercontent.com/22502305/95634958-0cea9780-0a59-11eb-9687-e7ee189b4669.png)

**Comment by katonage** [comment]
I find it unintuitive to store the 'default' flag per device in the lovelace dashboards config, and in general I have concerns with this 'default' flag what does it means? 
*first dashboard to load on boot
*automatically generated dashboard
*seen in the menu (overview...) 

I would make these behaviors separate and obvious. 

*I'd create a toggle in the config / lovelace to control autogeneration (read only dashboard layout) (global setting, can be set to any dashboard )
*Menu presence should be (only) controlled in the menu editor ( per device)
*First dashboard to load: I would store the last viewed one for each device (per device), or optionally add a global flag to enforce one of the dashboards.
Like this the current 'default' dashboard would keep the same appearance in the config as standard ones, having 'admin only' flag too. And receiving a massive warning before switching on autogeneration for an already edited dashboard. 
Importantly not mixing per device and global config elements in the same (globally looking) lovelace config panel.

**Comment by zsarnett** [comment]
We are working on redefining the auto generated dashboard <https://github.com/home-assistant/frontend/discussions/7406>

**Comment by tomershvueli** [comment]
Would love to see this! I'm seeing similar behavior on the iOS app. It'd be great to be able to set a default custom dashboard, but still have access to the Overview auto-generated dashboard.
